logging/telemetryFilteredHook.go

```diff
@@ -19,30 +19,42 @@ package logging
 import (
 	"strings"
 
+	"github.com/pkg/errors"
```

Comment:
would you be able to use either "errors" package or `fmt.Errorf` ?
I don't see any other reference to this package, and I'd like to avoid adding new ones.
( unless there is a good reason for that, of course )

Reply:
fixed